charts/replicated-library/templates/lib/_collector_templates.tpl

```diff
@@ -0,0 +1,37 @@
+{{- define "replicated-library.troubleshoot.collector.template.cluster" -}}
+- {{ $ }}: {}
+{{- end }}
+
+{{- define "replicated-library.troubleshoot.collector.template.logs" -}}
+{{- if contains "kotsadm-" .ContextValues.logsCollector.appName -}}
+- logs:
+    collectorName: {{ .ContextValues.logsCollector.appName }}
+    name: kots/admin_console
+    selector:
+      - app={{ .ContextValues.logsCollector.appName }}
+{{- else if .ContextValues.logsCollector.appName -}}
+{{- $_ := set $.ContextNames "app" .ContextValues.logsCollector.appName -}}
```

Comment:
I have removed appName, just to use selector.
If user has 
```
- logs:
          collectorName: wg-easy
          selector:
            app: wg-easy
          namespace: default
          containerNames:
            - wg-easy
```
I am using `index $.Values.apps` to check `app: wg-easy`. If it has wg-easy, then I will use `include "replicated-library.labels.selectorLabels" $`. Else like kotsadm, it will be passed by user.